pyscriptjs/tests/unit/pyconfig.test.ts

```diff
@@ -135,4 +136,38 @@ describe('PyConfig', () => {
         instance.setAttribute("src", "/covfefe.json");
         instance.connectedCallback();
     });
+    it('connectedCallback should call loadRuntimes', async () => {
```

Comment:
nitpick but a newline above this would be good to have

Reply:
good point thank you!